proprietary/tokens/src/common/ams/focus.tokens.json

```diff
@@ -1,7 +1,7 @@
 {
   "ams": {
     "focus": {
-      "outline-offset": { "value": "2px" }
+      "outline-offset": { "value": "{ams.border.width.md}" }
```

Comment:
It's not really a border width though, it's an offset. Both just happen to be 2px

Reply:
Oh wait, it’s not the outline itself. Changed to `0.125rem`.